galaxy_ng/app/tasks/synclist.py

```diff
@@ -125,19 +125,27 @@ def curate_synclist_repository(synclist_pk, **kwargs):
 
     namespaces = synclist.namespaces.filter().values_list("name", flat=True)
 
+    # FIXME: For 'exclude', we want to remove all version of the excluded collection
+    is_highest = True
+    if synclist.policy == 'exclude':
+        is_highest = False
```

Comment:
This whole block is not needed because `synclist.policy == 'exclude'` already evaluates to a boolean